template/Content/Program.cs

```diff
@@ -0,0 +1,25 @@
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Routing;
+
+namespace FeatherHttp.Templates.HelloWorld 
```

Comment:
This changes to the user's namespace right?

Reply:
correct when the user provides the -n this will change also
`dotnet new featherhttp -n My.Awesome.Project` will result in a new directory of that name and the namespace will change as well